### DISCUSSION TITLE
Somewhat Related

### DISCUSSION THREAD

**Discussion by RobSheppard1976** [cjww-development/gatekeeper#34] [General]
I stumbled across your repos from a Reddit post today, and your name was/is similar to this site:

https://blog.cjwdev.co.uk/

Are you the dev of those applications, specifically AD Tidy? Just curious what happened, as they appear abandoned these days?

Cheers!

**Comment by chrisjwwalker** [comment]
@RobSheppard1976 apologies for the delayed response. Thanks for your interest in my repos, if there's anything I can help with specifically around this repo, or my other ones, give us a shout! In terms of the blog you've mentioned though, I don't own that, so won't be able to help unfortunately.

**Reply by RobSheppard1976** [reply]
No worries, was just curious if you were the same dev as your names were similar. AD Tidy has helped me a lot in the past, so wanted to say thanks. But seems like the site has been abandoned. No worries, thanks!
